### ISSUE TITLE
Should I expect the single task label model has the same answer with multiple task label model with single task input?

### ISSUE THREAD

**Issue by SenWu** [HazyResearch/metal#30]
Hi,

I tried to use the same data I use for single task label model to multiple task label model with the same setting. Should I expect the same/similar result from these two settings?

Here is an example:

```
import pickle

with open("data/multitask_tutorial.pkl", 'rb') as f:
    Xs, Ys, Ls, Ds = pickle.load(f)

from metal.multitask import TaskHierarchy
task_graph = TaskHierarchy(cardinalities=[2], edges=[])

from metal.multitask import MTLabelModel
mt_label_model = MTLabelModel(task_graph=task_graph)

mt_label_model.train([Ls[0][0]], n_epochs=200, print_every=20, seed=123)
prob1 = mt_label_model.predict_proba([Ls[0][0]])

from metal.label_model import LabelModel
st_label_model = LabelModel(k=2, seed=123)
st_label_model.train(Ls[0][0], n_epochs=200, print_every=20)

prob2 = st_label_model.predict_proba(Ls[0][0])
```

prob1 is not the same as prob2.

**Comment by ajratner** [comment]
Hi @SenWu ,

Thanks for pointing this out!  I'll check it out soon

**Comment by ajratner** [comment]
Hi @SenWu ,

Sorry for the delay here!  The issue is just a small one having to do with passing trivial single-task "hierarchies" into the `TaskHierarchy` class.  I fixed the issue (#34 ), once tests pass will merge it in!
